src/modules/visits/services/visits.service.ts

```diff
@@ -274,44 +275,55 @@ export class VisitsService {
 			}
 		}
 
```

Comment:
@bertyhell Emile showed me the error but my first thought its if it is access Denied with accessType Folders FE should return accessFolderIds array with strings which does not happen. I think that is an issue on Fe side.